listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -688,15 +708,18 @@ export default class RecentListens extends React.Component<
                           if (e.key === "Enter") this.handleClickNewer();
                         }}
                         tabIndex={0}
+                        href={
+                          isNewerButtonDisabled
+                            ? undefined
+                            : `${window.location.origin}${window.location.pathname}?min_ts=${previousListenTs}`
```

Comment:
would this be clearer if it was just `<a href="?min_ts=xxxxxxx">` ? I think the browser will sort out all of the relative path calculation properly.

Reply:
Mm, yes I suppose it should. Whatever I had in mind for deciding against it at first doesn't make sense. I'll change it now.